### DISCUSSION TITLE
Revised WaveContainer

### DISCUSSION THREAD

**Discussion by kaischallenberg** [ppy/osu#14191] [Ideas]
Short and painless:
Does it make sense to work on the WavesContainer or is the animation in the current version already final and does not need any further changes?

My suggestion:
The animation is a little longer and more appropriate to the sound.

Current Build:
https://user-images.githubusercontent.com/26031319/128784612-d58e08ea-2e98-4436-9cdf-44baeb4864d7.mp4

Suggestion:
https://user-images.githubusercontent.com/26031319/128784653-d03bdaaa-d48a-403b-b123-8902fc50ef4b.mp4

**Comment by peppy** [comment]
it looks worse IMO
